massa-network-worker/src/node_worker.rs

```diff
@@ -27,14 +31,21 @@ use tokio::{
 };
 use tracing::{debug, trace, warn};
 
+#[derive(Debug, Serialize, Deserialize)]
+pub enum ToSend {
+    Msg(Message),
```

Comment:
Clippy warning. Do we want to address it ?

Reply:
Do you think it's a problem, and if so, do you have a suggestion on have to address it?

Reply:
I don't if use box as clippy wants its a good idea maybe add attribute to clear the warning

Reply:
How about we leave it as a warning(as a reminder), and address it if it becomes a performance problem?

Reply:
Ok